src/Render.cc

```diff
@@ -16,11 +16,15 @@ using namespace snowcrash;
 namespace drafter {
 
     // JSON content-type regex
-    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json";
+    const char* const JSONRegex = "^[[:blank:]]*application/(.*\\+)?json[[:blank:]]*(;.*|$)";
+    const char* const JSONSchemaRegex = "^[[:blank:]]*application/schema\\+json[[:blank:]]*(;.*|$)";
```

Comment:
Please correct me if I am reading the regex incorrectly. Beginning of the line, possibly white space; exact string `application/shcema+json`;possibly white space and then either semicolon and some string and or end of line. So you are looking for exact substring. True the regex would fail on string like `asdf application/schema+json` but the substring will still match but is it likely to happen? Anyway this is just a discussion to improve my regex skills.